src/server.py

```diff
@@ -487,33 +476,26 @@ def run_seta_details_query():
         if head_rev or pushlog_id:
             priority = valve(head_rev, pushlog_id, branch, priority)
         else:
-            priority = None
+            priority = 0
 
     alljobs = JOBSDATA.jobtype_query()
 
     # Because we store high value jobs in seta table as default,
     # so we return low value jobs, means no failure related with this job as default
-    # when the priority is 0, otherwise we return high value jobs(1 ~ 5).
+
+    # priority = 0; run all the jobs
     if priority == 0:
+        jobtype = alljobs
+    # priority =5 run all low value jobs
+    elif priority == 5:
         low_value_jobs = [low_value_job for low_value_job in alljobs if
                           low_value_job not in jobtype]
         jobtype = low_value_jobs
+    # priority =1, run all high value jobs
+    elif priority == 1:
+        pass # use jobtype as a high value query
 
```

Comment:
I was thinking it would be different:
1 == detects 4 or more failures
2 == detects 3 failures
3 == detects 2 failures
4 == only 1 failure
5 == all jobs

right now that maps to:
1 == high value jobs
5 == low value jobs + high value jobs

this needs to be more robust and documented.  The ideal plan is to switch to this model:
select counter from taskrequests where branch = branch;
jobs_to_run = runnable_jobs.filter(priority=1)
if counter % 2 == 0:
   jobs_to_run += runnable_jobs.filter(priority=2)
if counter % 3 == 0:
   jobs_to_run += runnable_jobs.filter(priority=4)
if counter % 4 == 0:
   jobs_to_run += runnable_jobs.filter(priority=4)
if counter % 5 == 0:
   jobs_to_run += runnable_jobs.filter(priority=5)

this would then mean we don't run all the tests at once.  This also ignores the timeout clause- I am sure we can get there.